### PR TITLE
add Antrea description

### PR DESCRIPTION
**Pull request by yasensim**
can we backport it to 1.16 and 1.17? or should I send separate PRs?

### CODE REVIEW

**Comment by neolit123** [comment]
/lgtm
merge is up to sig-docs as per the policy discussion for third party content.
https://github.com/kubernetes/enhancements/pull/1327

File: content/en/docs/concepts/cluster-administration/networking.md [hunk 1]
```diff
@@ -98,6 +98,11 @@ AOS supports the use of common vendor equipment from manufacturers including Cis
 
 Details on how the AOS system works can be accessed here: http://www.apstra.com/products/how-it-works/
 
+### Antrea
```
Comment (sftim) [inline content/en/docs/concepts/cluster-administration/networking.md]: Antrea sorts before AOS, right?

Comment (yasensim) [reply content/en/docs/concepts/cluster-administration/networking.md]: yes, I will fix it in a moment

**Comment by sftim** [comment]
@Rajakavitha1 can you add a comment explaining the hold? (hope that's OK?)

**Comment by Rajakavitha1** [comment]
> @Rajakavitha1 can you add a comment explaining the hold? (hope that's OK?)

Sure!!! I put on hold because:
merge is up to sig-docs as per the policy discussion for third party content.
kubernetes/enhancements#1327

**Comment by jimangel** [comment]
antrea and open vswitch are open source and apache 2.0 licensed. I think this is acceptable content and should be merged into master. @neolit123 is correct, we don't really back-port fixes - however dev-1.16 will match what's in master at the time of the release as we roll into 1.17.

/lgtm

I'll leave the final approval for @zacharysarah!

**Comment by yasensim** [comment]
Thanks everyone for the comments and the approval! Can we do /hold cancel
?

**Pull request merged**